jack-core/src/com/rapleaf/jack/transaction/DbMetricsImpl.java

```diff
@@ -118,7 +118,7 @@ public String getSummary() {
     summary += String.format("\nAverage number of Active connections : %,.2f", getAverageActiveConnections());
     summary += ("\nTotal number of queries/executions : " + getTotalQueries());
     summary += ("\nConnections created : " + getCreatedConnectionsCount());
-    summary += String.format("\n Max capacity time (%%) : %,.2f %%", 100 * getMaxConnectionsProportion());
+    summary += String.format("\n Max capacity time (%%) : %,.2f %%", 100 * getMaxCapacityProportion());
```

Comment:
I noticed that there are still some format inconsistency. For example, there is an extra space before the "Max". That's also the case for `TransactorMetricsImpl` line 81 (before "Average"). Can you fix them?